src/navigation/navigation.cc

```diff
@@ -159,6 +159,19 @@ void Navigation::Initialize(const NavigationParameters& params,
   planning_domain_ = GraphDomain(map_file, &params_);
   initialized_ = true;
   sampler_->SetNavParams(params);
+
+  PathEvaluatorBase* evaluator;
```

Comment:
To make it easier to debug pointer related operations, pointers should be initialized to null:
```
PathEvaluatorBase* evaluator = nullptr;
```